api/src/api/members.js

```diff
@@ -72,6 +72,74 @@ router.post(
   }),
 );
 
+// Gives the enum options to the frontend to populate the dropdown.
+// The key in the 'options' object must be the same as the DB attribute.
+router.get(
+  '/options',
+  requireRegistered,
+  errorWrap(async (req, res) => {
+    const options = {
+      level: Member.levelEnum,
+      location: Member.locationEnum,
+      role: Member.roleEnum,
+      gradSemester: Member.semesterEnum,
+      generationSemester: Member.semesterEnum,
+      classStanding: Member.classStandingEnum,
+      status: Member.statusEnum,
+    };
+
+    // Label all items with 'label' and 'value'
+    const labeledOptions = {};
+    for (var attributeLabel in options) {
+      if (!Object.prototype.hasOwnProperty.call(options, attributeLabel))
+        continue;
```

Comment:
I actually need to change this in `validateMemberQuery` too, but I just remembered a better way to loop through object keys, values, and entries. https://zellwk.com/blog/looping-through-js-objects/

So here I'd suggest `for (const [attributeLabel, optionsEnum] of Object.entries(options))`